### ISSUE TITLE
pypi version is old: "ImportError: cannot import name 'VatOption' from 'energyzero'"

### ISSUE THREAD

**Issue by sanderjo** [klaasnicolaas/python-energyzero#289]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Might be a PEBKAC, but the energyzero installed via pip has no VatOption

```
sander@zwart2204:~$ python3 -m pip install energyzero -U
Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: energyzero in ./.local/lib/python3.10/site-packages (0.5.0)
```

```
sander@zwart2204:~$ python3
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from energyzero import EnergyZero, VatOption
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'VatOption' from 'energyzero' (/home/sander/.local/lib/python3.10/site-packages/energyzero/__init__.py)
>>> 
```

```
>>> import energyzero
>>> energyzero.
energyzero.Electricity(                energyzero.EnergyZeroError(            energyzero.energyzero                  
energyzero.EnergyZero(                 energyzero.EnergyZeroNoDataError(      energyzero.exceptions                  
energyzero.EnergyZeroConnectionError(  energyzero.Gas(                        energyzero.models                      
>>> 
```

### Expected Behavior

working VatOption

### Steps To Reproduce

See above

### Environment

```Markdown
sander@zwart2204:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.3 LTS
Release:	22.04
Codename:	jammy
```

**Comment by klaasnicolaas** [comment]
That's because you're using Python 3.10 and you don't have the latest package version

**Comment by sanderjo** [comment]
> That's because you're using Python 3.10 

Correct

> and you don't have the latest package version

That's my point: I use the pypi version. So I would expect the latest version. 

Why isn't that the case? No available on pypi? Not available on pypi for Python 3.10? PEBKAC? ...?

**Comment by sanderjo** [comment]
Ah, workaround: installed python3.11 via deadsnakes, and install energyzero in that context:

```
sander@zwart2204:~$ python3.11 -m pip install energyzero
Defaulting to user installation because normal site-packages is not writeable
Collecting energyzero
  Downloading energyzero-2.1.0-py3-none-any.whl (10 kB)
```

and VatOption now is available:

```
sander@zwart2204:~$ python3.11
Python 3.11.7 (main, Dec  8 2023, 18:56:58) [GCC 11.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from energyzero import EnergyZero, VatOption
>>> 
```

Thanks.

**Comment by klaasnicolaas** [comment]
> Any reason for that? Does 2.1.0 use features not available in python 3.10?

This has to do with asyncio v3.9x in combination with the async_timeout lib, which didn't work properly from python 3.11 onwards and that's why I decided to drop support for 3.9 and 3.10 of python and add support for python 3.12.
